ceph-releases/hammer/ubuntu/16.04/base/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apt-get update &&  apt-get install -y wget unzip
 
 # Install Ceph
 RUN wget -q -O- 'https://download.ceph.com/keys/release.asc' | apt-key add -
-RUN echo deb http://ceph.com/debian-${CEPH_VERSION}/ trusty main | tee /etc/apt/sources.list.d/ceph-${CEPH_VERSION}.list
+RUN echo deb http://download.ceph.com/debian-${CEPH_VERSION}/ xenial main | tee /etc/apt/sources.list.d/ceph-${CEPH_VERSION}.list
```

Comment:
Wow, I am losing my mind.  I looked at the release page twice to verify...  right you are.